sklearn/cluster/tests/test_optics.py

```diff
@@ -1,814 +1,331 @@
-# Authors: Shane Grigsby <refuge@rocktalus.com>
-#          Adrin Jalali <adrin.jalali@gmail.com>
-# License: BSD 3 clause
-import warnings
+"""Testing for Spectral Clustering methods"""
```

Comment:
I think no changes from this file should appear in this PR. This might be a result of a merge conflict.